src/pet.js

```diff
@@ -0,0 +1,55 @@
+const MAXIMUM_FITNESS = 10;
+const DECREASE_FITNESS = 3;
+const INCREASE_FITNESS = 4;
+
+const MINIMUM_HUNGER = 0;
+const INCREASE_HUNGER = 5;
+const DECREASE_HUNGER = 3;
+
+const INCREASE_AGE = 1
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0; 
+    this.hunger = 0;
+    this.fitness = 10;
+};
+
+Pet.prototype = {
+    get isAlive () {
+   return this.age < 30 && this.hunger < 10 && this.fitness > 0
+}
+};
```

Comment:
The indentation is off here, each functions body should be indented for easier readability eg
```
Pet.prototype = {
    get isAlive () {
        return this.age < 30 && this.hunger < 10 && this.fitness > 0;
    }
};
```

Also going a bit further this could be made more readable by putting each statement on a new line:
```suggestion
Pet.prototype = {
    get isAlive () {
        return this.age < 30
            && this.hunger < 10
            && this.fitness > 0;
    }
```